web/static/js/app.jsx

```diff
@@ -217,7 +213,7 @@ var SearchBox = React.createClass({
     }
   },
 
-  onChange: function(e) {
```

Comment:
My thought is that it's worth it to remove unused arguments (even if they are meaningful) since in return you get notified when you accidentally forget to use an argument, which may help you catch a bug.

Your reasoning is that the `e` argument helps remind the reader that this is an event callback?